VocaDbWeb/Scripts/Stores/Song/SongDetailsStore.ts

```diff
@@ -308,7 +308,7 @@ export class SongDetailsStore {
 
 		this.tagUsages = new TagListStore(data.tagUsages);
 
-		if (data.songType !== SongType.Original && !this.originalVersion.entry) {
+		if (!this.originalVersion.entry) {
 			this.getOriginal(data.linkedPages);
 		}

```

Comment:
This will cause original songs that have unofficial links to another original song on VocaDB (or UtaiteDB/TouhouDB) to have the latter song considered the original version (this would only be shown in the UI and wouldn't be reflected in the database).

Example: https://vocadb.net/S/201986 (this one actually doesn't matter in the end as the song will have a properly defined demo version).

Could also be checked with an SQL query to the database.